src/Jackett.Common/Indexers/Nordicbits.cs

```diff
@@ -293,86 +294,77 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
                 // Getting results & Store content
                 var response = await RequestStringWithCookiesAndRetry(request, ConfigData.CookieHeader.Value);
-                _fDom = response.Content;
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(response.Content);
 
                 try
                 {
-                    var firstPageRows = FindTorrentRows();
-
-                    // Add them to torrents list
-                    torrentRowList.AddRange(firstPageRows.Select(fRow => fRow.Cq()));
+                    var firstPageRows = FindTorrentRows(dom);
 
                     // If pagination available
                     int nbResults;
                     int pageLinkCount;
-                    nbResults = 1;
                     pageLinkCount = 1;
 
                     // Check if we have a minimum of one result
-                    if (firstPageRows.Length > 1)
+                    if (firstPageRows != null && firstPageRows.Length > 1)
```

Comment:
Merge sequential checks
```suggestion
                    if (firstPageRows?.Length > 1)
```